src/Controller/UserController.php

```diff
@@ -117,15 +117,19 @@ public function deleteUser(Request $request, array $requestAttributes): Response
 
```

Comment:
From what I can see now you can filter users by role. As the number of filters grows(including the possibility of multiple filters at once. the code in this method will become tedious.
If isset($rA['some_filter]) x 5 do smth. 
In a future iteration I would extract the creation of $filters in a separate function. 
I would also consider mapping filters together inside $requestAttributes if  that were possible. 

This is not a change request. Just my 2 cents.

Reply:
Noted! Thank you!